subcommand/get-consul-client-ca/command.go

```diff
@@ -0,0 +1,227 @@
+package getconsulclientca
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/consul/command/flags"
+	"github.com/hashicorp/consul/lib"
```

Comment:
Yeah, good point. I'll update it to say consul-k8s rather than Consul since that is the "product" that's making the go-discover call. That way we can also use the version of consul-k8s rather than Consul, which is easier to retrieve.